sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusExtensionConfigProvider.cs

```diff
@@ -81,6 +81,7 @@ public void Initialize(ExtensionConfigContext context)
             context
                 .AddConverter(new MessageToStringConverter())
                 .AddConverter(new MessageToByteArrayConverter())
+                .AddConverter<ServiceBusReceivedMessage, BinaryData>(message => message.Body)
```

Comment:
The implementation is more complicated than in EH, because we have this separate ServiceBusAttributeBindingProvider type.

Also the semantics are slightly different between the two services and perhaps it should be rationalized (though it is consistent with T1):
- In SB, both string and BinaryData output argument that is null, will NOT send a ServiceBusMessage
- In EH, null string and BinaryData will send an empty EventData

/cc @paulbatum